versions.lock

```diff
@@ -1,172 +1,175 @@
 com.palantir.safe-logging:logger-spi:3.0.0 (2 constraints: 0b1e2f7a)
 com.palantir.safe-logging:preconditions:3.0.0 (18 constraints: 1620cf83)
 com.palantir.safe-logging:safe-logging:3.0.0 (17 constraints: fc0dd43b)
-com.palantir.safethreadlocalrandom:safe-thread-local-random:0.1.0 (3 constraints: 933225f9)
 com.palantir.tokens:auth-tokens:3.14.0 (4 constraints: fb421419)
-com.palantir.tracing:tracing:6.12.0 (8 constraints: 6c7d7f30)
-com.palantir.tracing:tracing-api:6.12.0 (5 constraints: 904fd1f5)
-com.palantir.tracing:tracing-jersey:6.12.0 (1 constraints: 3b054d3b)
-com.palantir.tracing:tracing-okhttp3:6.12.0 (1 constraints: 3b054d3b)
+com.palantir.tracing:tracing:6.13.0 (8 constraints: 7b80c354)
+com.palantir.tracing:tracing-api:6.13.0 (5 constraints: 9e527e9d)
+com.palantir.tracing:tracing-jersey-jakarta:6.13.0 (1 constraints: 3c05503b)
+com.palantir.tracing:tracing-okhttp3:6.13.0 (1 constraints: 3c05503b)
 com.palantir.tritium:tritium-api:0.51.0 (2 constraints: 2d1f6cbc)
 com.palantir.tritium:tritium-core:0.51.0 (1 constraints: 3e103fa2)
 com.palantir.tritium:tritium-metrics:0.51.0 (5 constraints: 054d2158)
 com.palantir.tritium:tritium-registry:0.51.0 (6 constraints: 425dead1)
 com.squareup.okhttp3:logging-interceptor:3.13.1 (1 constraints: 3a053f3b)
-com.squareup.okhttp3:okhttp:3.13.1 (6 constraints: 8459d8f9)
+com.squareup.okhttp3:okhttp:3.13.1 (6 constraints: 5151cdf6)
 com.squareup.okio:okio:1.17.2 (1 constraints: 850cc309)
 com.squareup.retrofit2:converter-jackson:2.4.0 (1 constraints: 08050136)
 com.squareup.retrofit2:retrofit:2.4.0 (2 constraints: 4b1d8704)
+com.sun.activation:jakarta.activation:2.0.1 (1 constraints: 801dd0e8)
 com.thoughtworks.paranamer:paranamer:2.8 (1 constraints: 7b15b5d4)
-commons-codec:commons-codec:1.15 (1 constraints: 0d13c328)
-io.dropwizard.metrics:metrics-core:3.2.6 (6 constraints: 8d5fe66f)
-jakarta.activation:jakarta.activation-api:1.2.2 (2 constraints: ba28fcbd)
-jakarta.annotation:jakarta.annotation-api:1.3.5 (3 constraints: e5315748)
-jakarta.el:jakarta.el-api:3.0.3 (1 constraints: fe135465)
-jakarta.validation:jakarta.validation-api:2.0.2 (3 constraints: 8e372576)
-jakarta.ws.rs:jakarta.ws.rs-api:2.1.6 (12 constraints: b0d52df8)
-jakarta.xml.bind:jakarta.xml.bind-api:2.3.3 (2 constraints: 3d33b287)
-joda-time:joda-time:2.10.8 (2 constraints: b4238380)
-org.apache.httpcomponents.client5:httpclient5:5.1.3 (1 constraints: cd13956e)
-org.apache.httpcomponents.core5:httpcore5:5.1.4 (3 constraints: 6139a70d)
-org.apache.httpcomponents.core5:httpcore5-h2:5.1.3 (1 constraints: 3d13093c)
-org.checkerframework:checker-qual:3.24.0 (3 constraints: ae1f0252)
-org.glassfish:jakarta.el:3.0.3 (2 constraints: bf231f5c)
-org.glassfish.hk2:hk2-api:2.6.1 (1 constraints: 6e0c620c)
-org.glassfish.hk2:hk2-locator:2.6.1 (1 constraints: 4410e0ad)
-org.glassfish.hk2:hk2-utils:2.6.1 (2 constraints: 21173277)
+io.dropwizard.metrics:metrics-core:4.2.10 (22 constraints: 035b00eb)
```

Comment:
this bumps our metrics-core dependency since it was conflicting all over the place at runtime.